### PR TITLE
Add simple 3- and 5-node integration testing

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by otoolep** [comment]
Build failed due to `go vet` issue, fixed now.

**Comment by otoolep** [comment]
@corylanou

File: cmd/influxd/server_single_integration_test.go [hunk 1]
```diff
@@ -19,59 +20,63 @@ import (
...
 		}
-		err = os.RemoveAll(tmpDataDir)
-		if err != nil {
-			t.Logf("Failed to clean up %q: %s\n", tmpDataDir, err)
-		}
-	}()
 
-	if s == nil {
-		t.Fatalf("Failed to open server")
+		time.Sleep(3 * time.Second)
```
Comment (otoolep) [inline cmd/influxd/server_single_integration_test.go]: @corylanou says I should not need this.

**Comment by otoolep** [comment]
Green builds, with explicit port ranges for each cluster test. Ideally we'd have a way to shut down the HTTP TCP listeners.

**Comment by corylanou** [comment]
Looks good.  :ship:

**Comment by otoolep** [comment]
Thanks @corylanou

**Pull request merged**